core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -1215,6 +1215,10 @@ public int getScanThreshold() {
         return Integer.parseInt(getOptional("kylin.query.scan-threshold", "10000000"));
     }
 
+    public boolean isQueryDuplicationDummpyResponseEnabled() {
```

Comment:
how about rename to a meaningful name? for example, we can call this feature 'lazy-query' then we use lazy-query-enabled as the name. more ideas?

Reply:
Good name. I'll change it.